### ISSUE TITLE
Grafana prometheus query "instant" checkbox becomes unchecked when dashboard is opened

### ISSUE THREAD

**Issue by jdevalk2** [grafana/grafana#14292]
Grafana does not hydrate provisioned Prometheus query with "Instant" as an instant query. It leaves the checkbox always unchecked.

### What Grafana version are you using?
Grafana 5.4.0 docker image

### What datasource are you using?
Prometheus Datasource

### What OS are you running grafana on?
Docker image in Kubernetes

### What did you do?
Create a dashboard with panel, connected to Prometheus Datasource
In this Panel created a Prometheus query, with "Instant" checkbox checked.
Save the JSON of the Dashboard, set the dashboard to auto-provision
Reboot Grafana, verify that the Dashboard has been provisioned
Open the Dashboard again, go to the query panel.
Observe that "Instant" is now unchecked and the query runs as though "Instant" was never checked

If I look at the JSON on disk the "Instant" is set to true.
If I look at the JSON in the "data" column inside the Grafana database, the "Instant" is set to true.
If I open the dashboard in Grafana, the Instant is not checked, not true.

If I recheck the Instant checkbox the dashboard runs as before. But due to it being auto-provisioned there is no way to save the dashboard other than export the JSON and reboot.

I haven't checked if the auto-provisioninng makes a difference, I directly being able to save a dasboard with "Instant" query.

### What was the expected result?
I expect the checkbox that was checked, and that the JSON specifies as true, to remain checked.

### What happened instead?
Checkbox became unchecked the moment the dashboard was opened.

**Comment by marefr** [comment]
Please include the json file you're provisioning.

**Comment by torkelo** [comment]
strange, unable to replicate here:
https://play.grafana.org/d/bWrTNREik/prometheus-instant?orgId=1&panelId=2&fullscreen&edit&tab=metrics

**Comment by jdevalk2** [comment]
I did some more investigation and the panel is a replicated / template panel, replicated 3 times, one panel for a single Kubernetes namespace.

However it appears I changed the Instant setting in a "slave" copy. The confusing part is that "slave" panel still saves it's full JSON and instant can be seen as true serialized in the JSON.   When the panels are recreated upon dashboard reload / redeploy (Grafana autoprovisioning), then the values of the "primary" panel are apparently overwriting the values of the "slave" panel.

Technically it's not a bug, but I do think the user experience could be a bit better by greying out controls on replicated panels so only the primary ones can be changed.
